.Lib9c.Tests/Action/Snapshot/transfer_asset.TransferCrystal.summary.verified.txt

```diff
@@ -7,13 +7,6 @@
     UpdatedAddresses: [
       99A06c2Bd71f0EAa8196fE45BE5ca424eE809733,
       C61C376693E6B26717C9ee9aCA5A2453028f6ffA
-    ],
-    ValidatorSet: {
-      Bencoded: []
-    }
-  },
-  UpdatedAddresses: [
-    99A06c2Bd71f0EAa8196fE45BE5ca424eE809733,
-    C61C376693E6B26717C9ee9aCA5A2453028f6ffA
-  ]
+    ]
+  }
```

Comment:
Is this snapshot change a specification?

Reply:
This is going to change continually I suppose.
- `ValidatorSet` is missing since it is supposed to be `null`. Previous snapshot was wrong.
- `UpdatedAddresses` has been removed from `IAccountStateDelta`.